media.emby.EmbyTheater.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "media.emby.EmbyTheater",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
```

Comment:
22.08?

Reply:
Will bump, I always assumed the latest runtime was unstable for some reason, but it seems I was wrong.